styles.css

```diff
@@ -0,0 +1,506 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+}
+
+html {
+  font-family: 'Crete Round', serif;
+}
+
+:root {
+  --black: #1c1a19;
+  --wine: #730d15;
+  --red: #ff6b00;
+  --gray: gray;
+  --gray2: #262626;
+  --white: #f4f5f0;
+}
```

Comment:
Please, kindly make sure there are only the colors that you used in the design of your page